aspnet-core/xml/ns-Microsoft.AspNetCore.Components.Server.Circuits.xml

```diff
@@ -1,6 +1,6 @@
 <Namespace Name="Microsoft.AspNetCore.Components.Server.Circuits">
   <Docs>
-    <summary>To be added.</summary>
+    <summary>Provides classes for implementing a circuit in Blazor Server. Not intended for use outside of the Blazor Server framework.</summary>
```

Comment:
This has types like CircuitHandler which we expect users to author. Maybe 

> Provides classes for implementing and working with circuits in a Blazor Server app.

Reply:
Good catch. Will fix.